utils/dbt_docs/get_dbt_package_versions.py

```diff
@@ -0,0 +1,95 @@
+import requests
+import base64
+import re
+import json
+
+#Set your PAT key so you get the 5000 calls per hour for the github api
+headers = {'Authorization': f"token <YOUR_PAT_TOKEN>"}
+
+# Functions to decode the contents and process into the format we want
+def process_package(packages):
+    decoded = [x for x in base64.b64decode(packages.json()['content']).decode().split('\n')[1:] if x != '']
+    req_pkgs = {}
+    for i in range(len(decoded)):
+        if i % 2 == 0:
+            pkg_name = decoded[i].split(' ')[-1]
+            versions_re = re.compile('version: (.*)') # If it is hard pinned to something this will fail.
+            req_versions = versions_re.findall(decoded[i+1])[0].replace('"', '').replace(',', '').replace('[', '').replace(']', '')
+            req_pkgs[pkg_name] = req_versions
+
+    return req_pkgs
+
+def process_versions(project):
+    dbt_versions_re = re.compile('require-dbt-version: (.*)')
+    try:
+        req_versions = dbt_versions_re.findall(base64.b64decode(project.json()['content']).decode())[0].replace('"', '').replace(',', '').replace('[', '').replace(']', '')
+    except IndexError:
+        req_versions = '>=1.3.0 <2.0.0' # If we haven't specified, just assume it needs a new version
+    return req_versions
+
+# List all packages and their snowplow github
+all_packages = [
```

Comment:
How will we remember to update this stuff?

Reply:
I'm going to write up a bit of an internal process around new dbt package releases, we have some stuff we already do such as release it on the hub, and generally make a docs page for it, but we'd also now have to add it here and in #264 , plus in the master component versions list so I'll put something in Notion.